plugins/periskop/config.d.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /**
+   * Configuration options for the periskop plugin
+   */
+  periskop: {
+    /**
+     * Integration configuration for the periskop servers
+     * @visibility frontend
+     */
+    locations: Array<{
```

Comment:
Might `servers` make sense as a key name to a casual reader? Unless this is a nomenclature that already is familiar to users of the product, then never mind!

Reply:
There is no specific nomenclature within the periskop project, although it is heavily inspired by prometheus.io and there they refer to `instance` - I will replace `locations` with `instances`, but do please let me know if you think this is a bad idea